packages/esm-admin-systemconfig-app/src/dashboard/index.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ExtensionSlot } from '@openmrs/esm-framework';
+
+import { LinkCard } from './card.component';
+
+import styles from './index.scss';
+
+export const SystemAdministrationDashbord = () => {
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.systemAdminPage}>
+      <div className={styles.breadcrumbsContainer}>
+        <ExtensionSlot extensionSlotName="breadcrumbs-slot" />
+      </div>
+      <div className={styles.cardsView}>
+        <LinkCard header={t('config', 'Configurations')} viewLink="/openmrs/admin">
```

Comment:
Ok, so the main comment here: This should be implemented as an extension slot with each app providing the link card as an extension.

Reply:
@ibacher I have made it an ExtensionSlot and so far used it in the `OCL` app and its working. I will push other PRs for other repos.